nvdaHelper/remote/winword.cpp

```diff
@@ -351,6 +332,39 @@ bool collectCommentOffsets(IDispatchPtr pDispatchRange, vector<pair<long,long>>&
 	return !commentVector.empty();
 }
 
+  bool collectRevisionOffsets(IDispatchPtr pDispatchRange, vector<tuple<long,long,long>>& revisionsVector) {
+	IDispatchPtr pDispatchRevisions=NULL;
+	if(_com_dispatch_raw_propget(pDispatchRange,wdDISPID_RANGE_REVISIONS,VT_DISPATCH,&pDispatchRevisions)!=S_OK||!pDispatchRevisions) {
+		return false;
+	}
+	long iVal=0;
+	_com_dispatch_raw_propget(pDispatchRevisions,wdDISPID_REVISIONS_COUNT,VT_I4,&iVal);
+	for(int i=1;i<=iVal;++i) {
+		IDispatchPtr pDispatchRevision=NULL;
+		if(_com_dispatch_raw_method(pDispatchRevisions,wdDISPID_REVISIONS_ITEM,DISPATCH_METHOD,VT_DISPATCH,&pDispatchRevision,L"\x0003",i)!=S_OK||!pDispatchRevision) {
+			continue;
+		}
+		long revisionType=0;
+		if(_com_dispatch_raw_propget(pDispatchRevision,wdDISPID_REVISION_TYPE,VT_I4,&revisionType)!=S_OK) {
+			continue;
+		}
+		IDispatchPtr pDispatchRevisionScope=NULL;
+		if(_com_dispatch_raw_propget(pDispatchRevision,wdDISPID_REVISION_RANGE,VT_DISPATCH,&pDispatchRevisionScope)!=S_OK||!pDispatchRevisionScope) {
+			continue;
+		}
+		long start=0;
+		if(_com_dispatch_raw_propget(pDispatchRevisionScope,wdDISPID_RANGE_START,VT_I4,&start)!=S_OK) {
+			continue;

```

Comment:
Just to make sure, don't we need to log anything when one of these continue  statements is reached, e.g. when we can fetch the type and start, but not the end? That might indicate that something is broken